tests/ElasticaBundle/DependencyInjection/ElasticaExtensionTest.php

```diff
@@ -128,4 +144,89 @@ public function testLoadWithoutLogger()
         $this->assertEquals(ElasticaLogger::class, $loggerDefinition->getClass());
         $this->assertNull($loggerDefinition->getArgument(0));
     }
+
+    public function testDefaultClientShouldBeAutowiredByDefault()
+    {
+        $this->ensureAutowiring();
+        $config = [
+            [
+                'clients' => [
+                    'default' => [
+                        'host' => '127.0.0.1',
+                        'port' => '9200',
+                    ],
+                    'another' => [
+                        'host' => '192.168.1.1',
+                        'port' => '9200',
+                    ],
+                ],
+            ]
+        ];
+        $this->extension->load($config, $this->container);
+        $this->assertTrue($this->container->has(Client::class));
+        /** @noinspection PhpUnhandledExceptionInspection */
```

Comment:
Please avoid PHPStorm specific annotations

Reply:
Removed, but PHPStorm now have plenty of warnings about code. It was added because I prefer to keep code clean from warnings I'm aware of to be able to see new ones